### ISSUE TITLE
container_cidr missing from openstack_inventory.json in newton+

### ISSUE THREAD

**Issue by alextricity25** [rcbops/rpc-openstack#1624]
##### What type of issue is this? Please tick the appropriate box.
- [x] bug - existing functionality that does not work for you as documented
- [ ] feature - new functionality that you would like to see added

##### What is the expected behavior?
https://bugs.launchpad.net/openstack-ansible/+bug/1649632
##### What is actually happening?
https://bugs.launchpad.net/openstack-ansible/+bug/1649632
##### What were you attempting to do when you encountered this error (give as much detail as possible)? 
https://bugs.launchpad.net/openstack-ansible/+bug/1649632
##### How can we reproduce this issue?
https://bugs.launchpad.net/openstack-ansible/+bug/1649632
##### How can we verify your issue is addressed?
https://bugs.launchpad.net/openstack-ansible/+bug/1649632
##### If you are using or know of a workaround please describe the necessary steps?
https://bugs.launchpad.net/openstack-ansible/+bug/1649632
##### What is your evaluation on the importance of this issue, and why?
https://bugs.launchpad.net/openstack-ansible/+bug/1649632
----------------------------------------------------------------------------------------------------------------------------
###### For each pull request, update the branch line to reference it.
- [ ] master

##### Upgrade impact. Consider the impact on moving between major, minor and patch releases.
- [x] This change does not impact upgrades
- [ ] This change addresses a potential upgrade impact
- [ ] The upgrade impact is being tracked by another issue.

The issue tracking the upgrade impact is:

##### Testing impact
- [ ] An issue/pull-request has been create or exists to track the testing gap, and is referenced below
- [ ] This change includes additional testing
- [x] This change is tested by the existing setup

Additional testing links:

##### Release note
- [ ] This change includes a release note.
- [x] This change does not require a release note.

##### Does your change require documentation?
- [ ] Yes, and I submitted an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues "Docs issues")
- [ ] Yes, and I contributed a documentation patch to [docs-rpc](https://github.com/rackerlabs/docs-rpc "docs-rpc")
- [x] No documentation required for this change

Additional documentation links (please add links if you have ticked yes):

**Comment by nrb** [comment]
A fix for master is proposed https://review.openstack.org/#/c/410331/

That will need to be merged before I look into doing one for Newton.

**Comment by nrb** [comment]
@evrardjp I think this bug would be the place to propose an alternate solution within the F5 script

**Comment by major** [comment]
Patch to master has merged: https://review.openstack.org/#/c/410331/
Newton backport is merging now: https://review.openstack.org/#/c/410397/

**Comment by nrb** [comment]
There seems to be some sort of git error with the release notes on the Newton branch. I can't replicate locally, and it didn't appear on master.

**Comment by major** [comment]
@nrb Well I'm the doofus that messed with a release note in a backport. :disappointed:

**Comment by major** [comment]
It it breaks again, I'll revert to your original patch there, merge it, and fix the reno in a new patch. ;)

**Comment by nrb** [comment]
Currently blocked on https://bugs.launchpad.net/reno/+bug/1649982 and reno 1.9.0 getting rolled out to infra.

**Comment by nrb** [comment]
@alextricity25 https://review.openstack.org/#/c/410397/ has merged.

**Comment by alextricity25** [comment]
UPDATE: Waiting on https://review.openstack.org/#/c/416946/ to merge before we bump the SHA here in RPCO. Once that merges I'll bump the SHA in RPCO.
